SpringMVCAnnotationOnlineStore/src/main/java/org/o7planning/springmvconlinestore/entity/Order.java

```diff
@@ -24,29 +25,28 @@
 import javax.persistence.TemporalType;
 import javax.persistence.UniqueConstraint;
 
+import org.o7planning.springmvconlinestore.model.OrderInfo;
+
 @Entity
 @Table(name = "Orders") 
 public class Order implements Serializable {
 
     /**
 	 * Maximal Constructor
+     * 
 	 */
 	public Order(Date orderDate, String orderStatus, int amount, Double prodRetailPrice,
 			String customerName, String customerAddress, String customerEmail, String customerPhone,
 			String nameEngraving, String birthstoneID, String charmId1, String charmId2, String charmId3,
 			String charmId4, Product product, Customer customer) {
 		super();
 		
-		
-		DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
-		Calendar today = Calendar.getInstance();
-		dateFormat.format(today);
-
-		this.orderDate = String.valueOf(today.getTime());
+		Date date = new Date();
+		this.orderDate = date;
```

Comment:
Note that the MySQL field is `datetime`: 'YYYY-MM-DD'

Reply:
That's perfect- 100% compatible.